### PR TITLE
Implement Resolver Detail

### PR DESCRIPTION
**Pull request by josemigallas**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-7724

## Description
This is a work in progress, do not merge and do not review exhaustively.

### CODE REVIEW

File: ui/components/resolvers/ResolversContainer.js [hunk 1]
```diff
@@ -24,24 +25,25 @@ class ResolversContainer extends Component {
...
             <React.Fragment>
                 <CommonToolbar
                     buttons={this.getToolbarButtons()}
                     onFilter={name => this.setFilter(name)}
                 />
                 <div className={styles.flexWrapper}>
-                    <div className={styles.left}>
-                        <ResolversList onClick={this.onResolverClicked} />
+                    <div className={styles.resolversListContainer}>
+                        <ResolversList onClick={res => this.onResolverClicked(res)} />
```
Comment (pb82) [inline ui/components/resolvers/ResolversContainer.js]: This passes you three arguments and should look like: `(type, field, resolver) => this.onResolverClicked(type, field, resolver)`

Comment (josemigallas) [reply ui/components/resolvers/ResolversContainer.js]: It's being taken care of https://github.com/aerogear/data-sync-ui/pull/80/files#diff-46ef7d14318a2e0fa30f96b90956812dR10.

File: ui/components/resolvers/ResolverDetail.js [hunk 2]
```diff
@@ -11,10 +42,147 @@ class ResolverDetail extends Component {
...
+                        bsStyle="primary"
+                        onClick={() => this.save()}
+                    >
+                        Save
+                    </Button>
+                    <Button
+                        bsStyle="default"
+                        onClick={() => this.cancel()}
+                    >
+                        Cancel
```
Comment (pb82) [inline ui/components/resolvers/ResolverDetail.js]: What does the cancel button do? There is no dialog to close here.

File: ui/components/resolvers/Security.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
```
Comment (pb82) [inline ui/components/resolvers/Security.js]: It's ok to have this component in the repo but we whould not display it anywhere yet.

**Pull request merged**